ruleset/decoders/0575-eset-remote_decoders.xml

```diff
@@ -1,8 +1,26 @@
-<!-- Sample logs
-May  6 10:59:37 XXXXX ERAServer[5032]: {"event_type":"Audit_Event","ipv4":"XXX.XXX.XXX.XXX","hostname":"XXXXX","source_uuid":"9416183d-3XX3-4776-9783-9532a3a027bb","occured":"06-May-2021 09:59:37","severity":"Information","domain":"Domain group","action":"Login attempt","target":"a49d257e-ecc6-4063-95c6-5eb5e6b3e5df","detail":"Authenticating domain user 'XXXXXXXX'.","user":"","result":"Success"}
+<!--
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
+
+<!-- 
+  ESET decoders for:
+    ESET Remote Administrator v8.1
+  
+  Comments:
+    These decoders accepts logs generated by the ESET Remote Administrator and decodes them.
+  
+    The logs from ESET are generated in BSD format, so to achieve log decoding,
+    in the ESET Protect Console Syslog output format has to be configured.
+  
+    The supporting reference for configuring the ESET logs to output as Syslog is below.
+    Reference: https://help.eset.com/era_admin/65/en-US/admin_server_settings_syslog.html
+
+  Sample logs:
+    May  6 10:59:37 XXXXX ERAServer[5032]: {"event_type":"Audit_Event","ipv4":"XXX.XXX.XXX.XXX","hostname":"XXXXX","source_uuid":"9416183d-3XX3-4776-9783-9532a3a027bb","occured":"06-May-2021 09:59:37","severity":"Information","domain":"Domain group","action":"Login attempt","target":"a49d257e-ecc6-4063-95c6-5eb5e6b3e5df","detail":"Authenticating domain user 'XXXXXXXX'.","user":"","result":"Success"}
+-->
+
 <decoder name="eset-bsd">
-  <program_name>ERAServer</program_name>
+  <program_name>^ERAServer$</program_name>
```

Comment:
Are you sure about this change?

Reply:
I mean, no version number or anything like that can come appended to program name?